requirements.txt

```diff
@@ -0,0 +1,2 @@
+presidio_analyzer
+presidio_anonymizer
```

Comment:
please add version to these packages? you can look at the version on your local machine like that:

`pip freeze | grep presidio_analyzer`